docs/user/document/tracking/revision_history-usage.en.md

```diff
@@ -1,15 +1,19 @@
 # Revision History - Usage
 
-* The revision items sorted by date must have `number` values in ascending order.
+The revision history is used to track the development of the document in at least all published versions.
+There must be an entry in the revision history for each published version.
+For pre-release versions there are no entries in the version history. However, before the first release, the field can be used to record changes.
 
-* The `number` of the revision item with the most recent `date` value must be the same as in `/document/tracking/version`.
-  For this comparison build metadata is ignored.
-  If the document status is `draft` any pre-release part is also ignored.
+The revision items sorted by date must have `number` values in ascending order.
 
-* The revision items sorted by date must not omit a version `number`.
-  In case of semantic versioning, this applies only to the major version.
-  This sorted list of version numbers must start with (Major) version 0 or 1.
+The `number` of the revision item with the most recent `date` value must be the same as in `/document/tracking/version`.
+For this comparison build metadata is ignored.
+If the document status is `draft` any pre-release part is also ignored.
 
-* The same version `number` must not be repeated for multiple revision items.
+The revision items sorted by date must not omit a version `number`.
+In case of semantic versioning, this applies only to the major version.
+This sorted list of version numbers must start with (Major) version 0 or 1.
```

Comment:
What does that depend on? => Document status draft/final

Reply:
updated